libs/core/src/lib/carousel/carousel-item/carousel-item.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CarouselItemComponent } from './carousel-item.component';
+
+describe('CarouselItemComponent', () => {
+  let component: CarouselItemComponent;
+  let fixture: ComponentFixture<CarouselItemComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CarouselItemComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CarouselItemComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
having few test scenarios will help.

Reply:
have included tests in carousel component for carousel items as well. removed this file.